tool/migrate_nook_watcher_data.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+import textwrap
+
+import firebase_util
+from core_data_modules.logging import Logger
+
+firebase_client = None
+log = None
+
+def strip_project(data):
+    for entry in data:
+        try:
+            del list(entry.values())[0]['project']
+        except Exception:
+            pass
```

Comment:
I'm always skeptical of catching exceptions and doing nothing with them...
What kind of exceptions are we likely to get here? that the project field doesn't exist? are they likely to be one-offs, or for entire collections? would it be worth logging it as a warning at least?

Reply:
Since there some earlier documents without the project property, deleting a non-existent property in a dictionary throws an exception. As per your suggestion, I've decided to define the specific exception and show a warning if it doesn't exist